dashboard/app/models/lesson_group.rb

```diff
@@ -19,6 +19,16 @@ class LessonGroup < ApplicationRecord
   belongs_to :script
   has_many :lessons
 
+  acts_as_list scope: :script, column: :position
```

Comment:
https://github.com/brendon/acts_as_list

what are your thoughts on whether to use this -- does it provide any validations? I only see one place that we are using it today: https://github.com/code-dot-org/code-dot-org/blob/2b8290298a60590bc93b4bcf3a664e2d74c50ff6/dashboard/test/models/script_test.rb#L211

it seems fine to use, but only if we are using any of the features offered by this library.